ui/src/apps/clusterinfo/RootComponent.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from 'react';
+import { Row, Col } from 'antd';
+import { HashRouter as Router } from 'react-router-dom';
+
+import { Component_panel, ComponentPanel, MonitorAlertBar } from './components';
+
+import client from '@/utils/client';
+
+const App = () => {
+  const [loading, setLoading] = useState(true);
+  const [cluster, setCluster] = useState({});
+
+  useEffect(() => {
+    client.dashboard.topologyAllGet().then(data => {
+      setCluster(data);
+      setLoading(false);
+    });
```

Comment:
You can add a more `then()` likes: `.then(res => res.data).then(cluster => { setCluster(cluster) })` to get the real cluster.